src/Native/Runtime/portable.cpp

```diff
@@ -180,36 +181,161 @@ COOP_PINVOKE_HELPER(String *, RhNewString, (EEType * pArrayEEType, int numElemen
 
 #endif
 #if defined(USE_PORTABLE_HELPERS)
+struct RawEEType // TODO: defined in common.h
+{
+    uint16_t    m_componentSize;
+    uint16_t    m_flags;
+    uint32_t    m_baseSize;
+    MethodTable* m_pBaseType;
+    uint16_t    m_usNumVtableSlots;
+    uint16_t    m_usNumInterfaces;
+    uint32_t    m_uHashCode;
+};
 
-#ifdef HOST_ARM
+// dummy object for aligning next allocation to 8 that supports Methodtable GetBaseSize (12),HasComponentSize (false)
```

Comment:
This should use the existing `g_pFreeObjectEEType`

Reply:
Thanks, removed in favour of `g_pFreeObjectEEType`